tests/utils.js

```diff
@@ -459,7 +464,12 @@ const prepServices = async (config) => {
   }
 
   await listenForApi();
-  config = config || await browser.getProcessedConfig();
+  if(config === 'mocha'){
+    await listenForApi();
+    await runAndLogApiStartupMessage('User contact doc setup', setUserContactDoc);
+    return;
+  }
```

Comment:
I'm not sure I understand why this is needed. Why do we need to `listenForApi()` again? 
Also, the last line in this file is to set the user contact doc. I think it should work just fine without this condition.